SCons/Defaults.xml

```diff
@@ -199,13 +195,24 @@ of each directory in &cv-CPPPATH;.
 <para>
 The list of directories that the C preprocessor will search for include
 directories. The C/C++ implicit dependency scanner will search these
-directories for include files. Don't explicitly put include directory
-arguments in CCFLAGS or CXXFLAGS because the result will be non-portable
-and the directories will not be searched by the dependency scanner. Note:
-directory names in CPPPATH will be looked-up relative to the SConscript
-directory when they are used in a command. To force
-&scons;
-to look-up a directory relative to the root of the source tree use #:
+directories for include files. 
+Do not put include directory directives
```

Comment:
There are performance reasons to do this, which do include wanting to avoid scanning. But really no need to change the wording here I don't think..

Reply:
"... unless you want to include scanning... " ?